FitnessForge/server/routes.ts

```
@@ -0,0 +1,352 @@
+} from "@shared/schema";
+
+const JWT_SECRET = process.env.JWT_SECRET || "fitforge-secret-key";
+
+interface AuthRequest extends Request {
+  userId?: number;
+}
+
+// Middleware to verify JWT token
+const authenticateToken = (req: any, res: any, next: any) => {
+  const authHeader = req.headers['authorization'];
+  const token = authHeader && authHeader.split(' ')[1];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)